README.md

```diff
@@ -133,7 +133,13 @@ It also verifies multiple deployment strategies like:
 - Using OpenShift quarkus extension and Docker Build strategy
 
 ### `http-advanced`
-Verifies Server/Client http_2/1.1, Grpc and http redirections. 
+Verifies Server/Client http_2/1.1, Grpc and http redirections.
+
+### `jaxrs`
```

Comment:
Do we want to keep just this central README, or have one for each module?

Reply:
We will keep this into the main readme because the description is too short to have his own README

Reply:
After team discussion: this is a short one, so let's leave it at that. No separate README required.